hub/views.py

```diff
@@ -258,6 +258,7 @@ def get_context_data(self, **kwargs):
             ).select_related("data_type")
             for item in data.all():
                 context["mp"][item.data_type.name] = item.value()
+            context["mp"]["appg_memberships"] = [item.value() for item in data.filter(data_type__name="appg_membership")]
```

Comment:
this will need to change to match the updated data_type__name from the import script.